City-of-Helsinki/drupal-helfi-kymp
# UHF-2828: remove Matomo modules

Removes Matomo modules from the site.

How to test:

- Install KYMP instance
- Checkout this branch: `git checkout UHF-2828_remove_matomo`
- Import config: `make drush-cim`
- Clear cache (just in case): `make drush-cr`
- Log in to the site and make sure "Matomo" and "Matomo reports" modules no longer exist
